### ISSUE TITLE
Clear cron job

### ISSUE THREAD

**Issue by ghost** [padolsey/cron.js#4]
Hello.
I need to delete cronJob object. How can i do it?

**Comment by knguyen2525** [comment]
I also need to do this.

**Comment by ghost** [comment]
I have create it from forked current cron.js. You can use it:https://github.com/farkhad-alizade/cron.js

**Comment by knguyen2525** [comment]
Thanks so much!

How would I stop it?

var job = new CronJob('*/10 \* \* \* \* *', function(){
console.log("please stop me");
});

I tried job.stop();

Thanks again

**Comment by ghost** [comment]
just call stop method. job.stop(); Exactly!

**Comment by knguyen2525** [comment]
Hmmm, getting a job.stop is not a function. 

var job = new CronJob('*/10 \* \* \* \* *', function(){
    console.log("running");
    job.stop();
});

Any ideas :(

**Comment by ghost** [comment]
are you sure u forked cron.js from my repository , not from @padolsey ?

**Comment by knguyen2525** [comment]
yes, I also ran through the cron.js code to make sure it had the stop logic.

**Comment by ghost** [comment]
Please, try this:
var job = new CronJob('*/10 \* \* \* \* *', function(){
console.log("running");
});
job.stop();

**Comment by knguyen2525** [comment]
Same issue, BUT I just manually set job.events = [] and job.isStopped = true and that did the trick!

Cheers!
